Jenkinsfile

```diff
@@ -92,10 +92,20 @@ node('linux && docker') {
       }
 
       stage('Deployment pipeline upload') {
-        lerna = readJSON file: 'lerna.json'
-        version = lerna.version
-        (minor, major) = (version =~ /^([^\.]*)\.[^\.]*/)[0]
-        sh "deployment-package package create --with-deploy --version ${version} --resolve COMMIT_HASH=${commitHash} --resolve MINORVERSION=${minor} --resolve MAJORVERSION=${major} || true"
```

Comment:
I removed the recent `--version ${version}` because there isn't a single version to pass anymore.

According to the [docs](https://tools-docs.dep.cloud.coveo.com/deployment-package/cli.html#cmdoption-deployment-package-package-create-version):
> If your project is versioned, the version in this package. Must be unique and all versions must be greater than the last. Will be included in the package name.

Say I pass one project's version, e.g. headless, and there are only changes to atomic, I expect the pipeline will error because the version will not be greater than the last.
Moreover, if one is trying to publish Atomic changes, seeing the headless version set to 1.x would not be very helpful.